addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -235,18 +235,15 @@
                                                readonly="state in ('sending', 'done')"/>
                                     </div>
                                     <field name="mailing_filter_count" invisible="1"/>
-                                    <div name="mass_mailing_domain" invisible="mailing_model_name == 'mailing.list'"
-                                        class="col-md-6 d-block">
-                                        <field name="mailing_domain" widget="domain" options="{'model': 'mailing_model_real'}" readonly="state in ('sending', 'done')"/>
-                                    </div>
                                 </div>
 
                                 <field name="mailing_model_name" invisible="1"/>
                                 <field name="mailing_on_mailing_list" invisible="1"/>
                                 <field name="mailing_model_real" invisible="1"/>
-                                <div invisible="mailing_on_mailing_list">
-                                    <field name="mailing_domain" widget="domain" options="{'model': 'mailing_model_real', 'foldable': true}"
-                                    readonly="state in ('sending', 'done')"/>
+                                <div name="mass_mailing_domain" invisible="mailing_on_mailing_list">
```

Comment:
Do we need the extra div then? Can't we put the invisible attribute on the field directly?

Reply:
Indeed we can remove it :+1: